examples/high_frequency/emc/busbar.py

```diff
@@ -137,9 +175,23 @@
     get_self_terms=False, get_mutual_terms=True, category="C"
 )
 
+# Specify the traces to display after solving the model.ACL - Reduced Matrix MR_1_Series
+
+data_reducedm1_plot_self = q3d.matrices[1].get_sources_for_plot(
```

Comment:
@tizianrot can we find easier to read variable names? I mean for data_reducedm1_plot_self and data_reducedm2_plot_self